Makefile.rule

```diff
@@ -97,7 +100,7 @@ BUILD_LAPACK_DEPRECATED = 1
 NO_WARMUP = 1
 
 # If you want to disable CPU/Memory affinity on Linux.
-NO_AFFINITY = 1
+#NO_AFFINITY = 1
```

Comment:
Did you intend to revert this here? It's been considered as a good default by several projects using OpenBLAS, cf. https://github.com/JuliaLang/julia/pull/9639#issuecomment-68912940.

Reply:
Yes, hopefully this was an inadvertent change, since it seems unrelated to this PR …

Processor affinity is a poor default for running on non-dedicated machines.

Reply:
Fixed by #1495 a minute ago (unless I made another blunder)